services/dns/zone.go

```diff
@@ -14,8 +15,11 @@ import (
 var zoneResourceName = "DNS Zone"
 
 func (c *Client) CreateZone(ctx context.Context, d *schema.ResourceData) (zoneResponse dns.ZoneResponse, responseInfo utils.ApiResponseInfo, err error) {
-	request := setZoneCreateRequest(d)
-	responseData, apiResponse, err := c.sdkClient.ZonesApi.ZonesPost(ctx).ZoneCreateRequest(*request).Execute()
+	// Must is a helper that wraps a call to a function returning (UUID, error) and panics if the error is non-nil.
+	uuidV4 := uuid.Must(uuid.NewV4())
+	zoneUuid := uuid.NewV5(uuid.NewV5(uuid.NamespaceURL, "https://github.com/ionos-cloud/terraform-provider-ionoscloud"), uuidV4.String())
```

Comment:
in my opinion writing `uuid.NewV5(uuid.NewV5(uuid.NamespaceURL, "https://github.com/ionos-cloud/terraform-provider-ionoscloud"), uuidV4.String())` every time you want a new ID is a bit repetitive

What do you feel about creating a very small pkg `internal/tfid/tfid.go` which has the sole responsibility of just creating V5 UUIDs with the namespaceURL already set ? And you could call it like `tfid.New()` for example

Reply:
@avirtopeanu-ionos sure, not sure why I didn't do that in the beginning.

Reply:
@avirtopeanu-ionos done in 5369bab15a6a1015bc488c1140a42af0283ed025.